.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: [ '2.6.x', '2.5.x', '2.4.x', '2.3.x' ]
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Ruby
+      uses: actions/setup-ruby@v1
+      with:
+        version: ${{ matrix.ruby }}
+    - name: Install dependencies
+      run: |
+        # As of 3 Jan 2019, Travis defaults to bundler 2.0 - https://docs.travis-ci.com/user/languages/ruby/#bundler-20
+        # Linguist still requires Bundler 1.x due to Licensed.
+        gem uninstall -v '>= 2' -ax bundler || true
+        gem install bundler -v '< 2'
+        # Fetch all commits/refs needed to run our tests.
+        git fetch origin master:master v2.0.0:v2.0.0 test/attributes:test/attributes test/master:test/master
+        # Replace SSH links to submodules by HTTPS links.
+        sed -i 's/git@github.com:/https:\/\/github.com\//' .gitmodules
+        # Fetch all grammar submodules
+        git submodule init
+        git submodule sync --quiet
+        script/fast-submodule-update
+        # Install Linguist gem dependencies
+        bundle install --jobs 4 --retry 3 --without debug
```

Comment:
Wouldn't a script be more appropriate for this? Something like script/pre-release or something

Reply:
What, just like in the script I'm deleting in this PR? :wink: I was in two minds about this part and you rumbled me. I'll go back to the script.

Reply:
Sorry. 😁 IMHO, configuration files are best kept short-and-sweet; 5 or more lines of shell-script generally warrant their own file, especially when commented. It's up to you, though; I didn't want to block this PR by bikeshedding, hence the review comment instead of a requested change.